tensorflow/python/ops/image_ops_impl.py

```diff
@@ -640,6 +640,11 @@ def transpose(image, name=None):
 
   Raises:
     ValueError: if the shape of `image` not supported.
+
+  Usage Example:
+    import tensorflow as tf
+    x = tf.random.normal(shape=(256, 256, 3))
```

Comment:
Make this a doctest (use >>>, add the outputs, and use a small example, like a 1x2 image, so people can understand what happens from reading the example)